examples/deepspeed/gpt_neox/det_utils.py

```diff
@@ -7,7 +7,7 @@
 from megatron.neox_arguments import NeoXArgs
 
 from determined.pytorch import MetricReducer, PyTorchCallback
-from determined.tensorboard.metric_writers.pytorch import TorchWriter
+from determined.pytorch.tensorboard import TorchWriter
```

Comment:
there's more cleanup to be done in this file; see the TensorboardWriter callback.